containers/claims/tests/test_default_route.py

```diff
@@ -0,0 +1,9 @@
+from fastapi.testclient import TestClient
+from ..main import app
+
+client = TestClient(app)
+
+
+def test_default_route():
+    response = client.get("/")
```

Comment:
Why create a fixture to not use it? 

```suggestion
def test_default_route(client_fixture):
    response = client_fixture.get("/")
```